src/main/resources/entity-types/circulation/simple_loan_policy.json5

```diff
@@ -123,17 +123,6 @@
       visibleByDefault: true,
       valueGetter: ":sourceAlias.jsonb->'loansPolicy'->>'profileId'",
     },
-    {
-      name: 'closed_library_due_date_management',
-      sourceAlias: 'lp',
-      dataType: {
-        dataType: 'stringType',
-      },
-      isIdColumn: false,
-      queryable: true,
-      visibleByDefault: false,
-      valueGetter: ":sourceAlias.jsonb->'loansPolicy'->>'closedLibraryDueDateManagementId'",
-    },
```

Comment:
Somehow, we ended up with multiple versions of this field, so I deleted one (I picked the one that wasn't referenced in MODFQMMGR-361)